packages/video_player/video_player/lib/video_player.dart

```diff
@@ -31,6 +31,9 @@ VideoPlayerPlatform get _videoPlayerPlatform {
   return currentInstance;
 }
 
+/// Signature of a callback that returns the current duration after scrubbing through the [VideoProgressIndicator].
+typedef OnScrubbingCallback = void Function(Duration)?;
```

Comment:
if you're already making this optional on the classes it's used in, is it still necessary to do it here as well?

Reply:
Adjusted